android/src/main/kotlin/com/airship/flutter/EmbeddedViewFactory.kt

```diff
@@ -0,0 +1,21 @@
+package com.airship.flutter
+
+import android.content.Context
+import io.flutter.plugin.common.BinaryMessenger
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.plugin.common.StandardMessageCodec
+import io.flutter.plugin.platform.PlatformView
+import io.flutter.plugin.platform.PlatformViewFactory
+
+class EmbeddedViewFactory(private val binaryMessenger: BinaryMessenger) : PlatformViewFactory(StandardMessageCodec.INSTANCE) {
```

Comment:
We should be able to include this class in the other file, like we do in Swift. If flutter doesn't like that for some reason, there's probably a compat annotation that will make it work. lmk if that's the case, I'm happy to help find the right one(s)